isis/tests/Fixtures.cpp

```diff
@@ -2,14 +2,26 @@
 
 namespace Isis {
 
+  void TempTestingFiles::SetUp() {
+    ASSERT_TRUE(tempDir.isValid());
+  }
+
+
   void DefaultCube::SetUp() {
+    TempTestingFiles::SetUp();
+
+    tempFile.setFileTemplate(tempDir.path() + "/XXXXXX.cub");
```

Comment:
Are the templates still necessary if the temp directory as a whole (files included) will be cleaned up or am I missing something?

Reply:
The template is required to put the file in the temp directory and by adding `.cub` we ensure that we only make one file.

Reply:
I don't see where in the docs it says the files under the temp dir need to be tempfile objs. Why not create the temp dir, save the path (or tempdir object) as a member of the fixure, and make the underlying file names match 1-1 with what is in the test/data/<fixture> area?

Reply:
The files in the tempdir don't need to be tempfiles also. I'll change these to just regular files.